app/controllers/conversations.js

```diff
@@ -432,7 +432,7 @@ class ConversationsController extends BaseController {
       {
         _id: { $in: usersIds.filter((el, i) => usersIds.indexOf(el) === i) },
       },
-      ["_id", "first_name", "last_name", "login", "email", "phone"],
+      includes || ["_id", "first_name", "last_name", "login", "email", "phone"],
```

Comment:
`includes ` will have `["id"]` value. Should we convert it to `["_id"]` ?

Reply:
You don't have to convert, it won't affect the result. But I will add the following to increase reliability in the future.